libs/s25main/desktops/dskGameInterface.cpp

```diff
@@ -689,7 +689,8 @@ bool dskGameInterface::Msg_KeyDown(const KeyEvent& ke)
         case KT_F9: // Readme
             WINDOWMANAGER.ToggleWindow(std::make_unique<iwTextfile>("readme.txt", _("Readme!")));
             return true;
-        case KT_F10: {
+        case KT_F10:
+        {
```

Comment:
I think for this there is an option to indent the case labels correctly

Reply:
What do you mean by "correctly"? They are correct aren't they? An opening brace is on a new line on the same level as the outer code and code inside braces is indented by 1 level. Now all is perfectly fine